web/css-part-2.md

```diff
@@ -0,0 +1,94 @@
+
+### Guided Practice
+1. Spend 30 minutes playing [CSS Diner](https://flukeout.github.io/). You play by selecting certain HTML-like dishes and foods using CSS Selectors.
+1. Spend 30 minutes playing [Flexbox Froggy](https://flexboxfroggy.com/).  This one is cute - you arrange frog on lily pads using flexbox.
+1. During Flexbox Froggy or after, take 5 minutes to look at this [Flexbox Cheatsheet](https://flexbox.malven.co/).  There are more good ones out there if you care to search.
+1. [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS) has great detailed documentation on single CSS concept. Spend 5 minutes scanning these examples:
+  - [MDN Color Picker Tool](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Colors/Color_picker_tool)
+  - [The Box Model](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model)
+  - [Sizing](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Sizing_items_in_CSS)
+  - [drop-shadow()](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/drop-shadow)
+  - [:active pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:active)
+  - [::before pseudo-element](https://developer.mozilla.org/en-US/docs/Web/CSS/::before)
+1. Although CSS issues tend to be relatively easy to Google, https://css-tricks.com/ has one of the best collections of **CSS Guides**.  Take 15 minutes total to look at some of the most popular articles. You don't need to read them throughly, but instead get a feel for some of the common problems you'll be solving.
+  - Flexbox: https://css-tricks.com/snippets/css/a-guide-to-flexbox/
+  - Centering: https://css-tricks.com/centering-css-complete-guide/
+  - Color: https://css-tricks.com/nerds-guide-color-web/
+  - Media Queries: https://css-tricks.com/a-complete-guide-to-css-media-queries/
+  - Links and Buttons: https://css-tricks.com/a-complete-guide-to-links-and-buttons/
+  - Position Property (video): https://css-tricks.com/video-screencasts/198-about-the-position-property/
+
+### Independent Practice
+
+No pairs will be assigned for this project, but you should rely on your classmates for help once you've spent 20 minutes working independently through something you're stuck on.
+
+The activity for the HTML, CSS, and JS DOM lessons is a project that allows you to combine your new skills in each of these areas.
+
+[Click here to access the instructions for the CSS portion of the recipe-page project](../projects/recipe-page/phase-2-css-prompt.md). Please don't move on to the prompts for the other topics yet.
+
+Check for [Inline CSS](https://www.codecademy.com/articles/html-inline-styles) contents while removing _style_ tag from the webpage.
+
+Tutorials for changing and removing style tags are provided in the [CSS slides](https://docs.google.com/presentation/d/1p-IXWxo0NEbZbHQ_Mdoo-A9dlFXqfSPOfW6navfyeTI/edit?usp=sharing)
+
+### Challenge
+
```

Comment:
If there is no challenge should we remove this section?

Reply:
good catch